example_flutter/lib/keyboard_test_page.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/services.dart';
+
+class KeyboardTestPage extends StatefulWidget {
```

Comment:
Please document at least the public class (which is an analyzer failure under the current settings).